cms/static/js/spec/views/xblock_spec.js

```diff
@@ -62,8 +62,8 @@ define(["jquery", "URI", "common/js/spec_helpers/ajax_helpers", "common/js/compo
                         mockCssUrl = "mock.css",
                         headHtml;
                     postXBlockRequest(requests, [
-                        ["hash1", { mimetype: "text/css", kind: "text", data: mockCssText }],
-                        ["hash2", { mimetype: "text/css", kind: "url", data: mockCssUrl }]
+                        ["xblock_spec_hash1", { mimetype: "text/css", kind: "text", data: mockCssText }],
```

Comment:
`module_edit_spec.coffee` is also using `hash1-6` and this results in a conflict when all tests are run.